client/my-sites/themes/auto-loading-homepage-modal.scss

```diff
@@ -0,0 +1,36 @@
+.themes__auto-loading-homepage-modal {
+	min-height: 90px;
+	max-width: 320px;
+	margin: 0 auto;
+
+	@include breakpoint( '<480px' ) {
+		box-sizing: border-box;
+		max-width: 100%;
+	}
+
+	+ .dialog__action-buttons {
+		text-align: center;
+
+		.button:first-child {
+			margin-left: 0;
+		}
+
+		@include breakpoint( '<480px' ) {
+			.button,
+			.button:first-child {
```

Comment:
You could avoid adding extra styles to the `:first-child` hereby isolating the cancellation fo the `margin-left` into a separate selector. Otherwise, you're increasing the specificity of the first button and adding unwanted additional styles which would be handled by the lower specificity of the main `button` selector.